verify-cborld-presentation/src/App.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019 - MATTR Limited
+ * All rights reserved
+ * Confidential and proprietary
+ */
+
+import React, { useState } from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import styled from "styled-components";
+
+import { HomePage, ScanPage, ResultPage, QrPage } from "./pages";
+
+export const App = (): React.ReactElement => {
+  const [qrCode, setQrCode] = useState<string>();
```

Comment:
Maybe we can store the qrCode into context?

Reply:
yeah agree would be good in general but thinking `useState` approach is simplest solution for this app

Reply:
Redux seems quite heavy. And tried `useContext` but it ended up being more code